srcpkgs/wtype/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'wtype'
+pkgname=wtype
+version=0.3
+revision=1
+archs="~*-musl"
+build_style=meson
+hostmakedepends="pkg-config wayland-devel"
+makedepends="libxkbcommon-devel wayland-devel"
+depends="libxkbcommon wayland"
```

Comment:
This line is not required. The dependencies should be detected automatically.

Reply:
okay I removed it